Lesson2/src/Homework5.java

```diff
@@ -0,0 +1,28 @@
+public class Homework5 {
+
+
+    public static void main(String[] args) {
+        double array[] = {1, 2, 3, 4, 5};
+
+
+        int sum = 0;
+        for (int i = 0; i < array.length; ++i) {
+            sum += array[i];
+
+
+        }
+        System.out.println (sum / array.length);
```

Comment:
you need to put this code in public static double average(int[] array) method (instead of lines 17,18). But with one difference - replace System.out.println (sum / array.length); to return sum / array.length;

and then here, in main method, you can call System.out.println(average(array));

one more thing to add:

variable sum - type int, array.length - type int. Result of sum / array.length will be int too.
To change this - you need to change type of sum variable to double